shardingsphere-mode/shardingsphere-mode-core/src/main/java/org/apache/shardingsphere/mode/metadata/MetaDataContextsFactory.java

```diff
@@ -79,4 +83,14 @@ private static DatabaseConfiguration createEffectiveDatabaseConfiguration(final
         Collection<RuleConfiguration> databaseRuleConfigs = persistService.getDatabaseRulePersistService().load(databaseName);
         return new DataSourceProvidedDatabaseConfiguration(effectiveDataSources, databaseRuleConfigs);
     }
+    
+    private static Map<String, ShardingSphereDatabase> reloadDatabases(final Map<String, ShardingSphereDatabase> databases, final MetaDataPersistService persistService) {
+        Map<String, ShardingSphereDatabase> result = new ConcurrentHashMap<>(databases.size(), 1);
```

Comment:
`Collections.singletonMap()` may make this more elegant ？

Reply:
> `Collections.singletonMap()` may make this more elegant ？

I think you are right when i know the key and value explicitly. But no define value here.